metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfigDescriptor.java

```diff
@@ -122,6 +123,13 @@ private String getPropsUrl() {
         }
       }
     }
+    if (url == null) {
+      URL uri = MetricConfig.class.getResource("/" + MetricConstant.CONFIG_NAME);
+      logger.info("MetricConfig.class.getResource,{}", uri);
+      if (uri != null) {
+        return uri.getPath();
+      }
+    }
```

Comment:
Add some comments about this if-branch, actually I can't understand its motivation.